DDR4 OC Guide.md

```diff
@@ -436,35 +451,41 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
     | 3733 | 59728 |
     | 3800 | 60800 |
     | 4000 | 64000 |
-    * Your read and write bandwidth should be 90% - 95% of the theoretical maximum bandwidth.
-      * On single CCD Ryzen 3000 CPUs, write bandwidth should be 90% - 95% of half of the theoretical maximum bandwidth.  
+    
+    * Your read and write bandwidth should be 90% - 98% of the theoretical maximum bandwidth.
+      * On single CCD Ryzen 3000-5000 CPUs, write bandwidth should be 90% - 98% of half of the theoretical maximum bandwidth.  
         It is possible to hit half of the theoretical maximum write bandwidth. See [here](https://redd.it/cgc9bh).
+      * Percentage of theoretically max bandwidth is inversely proportional to most memory timings. Generally speaking, as RAM timings are tightened, this value will increase.
 
 1. I would recommend to tighten some of the secondary timings first, as they can speed up memory testing.  
    My suggestions:
    
    | Timing | Safe | Tight | Extreme |
    | ------ | ---- | ----- | ------- |
-   | tRRDS tRRDL tFAW | 6 6 24 | 4 6 16 | 4 4 16 |
+   | tRRDS tRRDL tFAW | 6, 6, 24 | 4, 6, 16 | 4, 4, 16 |
    | tWR | 16 | 12 | 10 |
-   * Minimum tFAW can be is tRRDS * 4.
+   * The minimum value for which lowering tFAW will have an effect on the performance of RAM is `tRRDS * 4` or `tRRDL * 4`, whichever is greater.
    * You don't have to run all of the timings at one preset. You might only be able to run tRRDS tRRDL tFAW at the tight preset, but you may be able to run tWR at the extreme preset.
    * On some Intel motherboards, tWR in the UEFI does nothing and instead needs to be controlled through tWRPRE (sometimes tWRPDEN). Dropping tWRPRE by 1 will drop tWR by 1, following the rule tWR = tWRPRE - tCWL - 4.
      
 2. Next is tRFC. Default for 8Gb ICs is 350**ns** (note the units).
    * Note: Tightening tRFC too much can result in system freezes/lock ups.
+   * tRFC is a timing regulating the number of cycles for which the dram capacitors are "recharged" or refreshed. Because capacitor charge loss is proportional to temperature, RAM operating at higher temperatures may need substantially higher tRFC values.
```

Comment:
```suggestion
   * tRFC is the number of cycles for which the DRAM capacitors are "recharged" or refreshed. Because capacitor charge loss is proportional to temperature, RAM operating at higher temperatures may need substantially higher tRFC values.
```